### ISSUE TITLE
update napalm ping response keys

### ISSUE THREAD

**Issue by jedelman8** [napalm-automation/napalm-base#240]
Reference: https://github.com/napalm-automation/napalm-ansible/pull/53

 one thing I'm thinking about within the ansible module and @dbarrosop sugessted maybe we just update the `ping` method within NAPALM so we can keep the ansible module a wrapper.

Normalizing the key used regardless if success or error.

Today it's this:

```
# when echo request succeeds
  "{"success": {"packet_loss": 0, "probes_sent": 2,
           "results": [{"ip_address": "10.0.0.5:", "rtt": 1.71},
            {"ip_address": "10.0.0.5:", "rtt": 0.733}],
            "rtt_avg": 1.225, "rtt_max": 1.718, "rtt_min": 0.733,
            "rtt_stddev": 0.493}}
   # when echo request fails
   {"error": "connect: Network is unreachable\n"}}
```
I'd rather see a status key that has success or error as values. Then a msg key for the error message and results for the dict of results. What are your thoughts?

**Comment by mirceaulinic** [comment]
I would just say to wait till OC will provide the models for action commands and provide a proper implementation. These methods have been designed improperly from the beginning and I think we'd only continue on the same line doing so.

**Comment by jedelman8** [comment]
Fair enough if that's the consensus...module works and my use-case has been accomplished...so for now, it'll remain a custom module!

**Comment by dbarrosop** [comment]
@mirceaulinic are there any OC models for actions that you know of? I haven't seen them (haven't looked for them either)

**Comment by mirceaulinic** [comment]
Currently there aren't any models available yet, but there will be something in the public repo soon-ish :-)

**Comment by dbarrosop** [comment]
Something tells me the model is going to be even messier than the platforms model...

**Comment by jemcek** [comment]
Are there any news regarding this issue? 

Currently I get completely different answers from the same unreachable destination using Junos and IOS what makes further processing more complicated. Also is the "probes_sent":0 and "packet_loss": -2 on IOS expected result in case 2 echo request were sent and no reply was received?

IOS:
"item": "4.4.4.4",
"results": {
        "success": {
            "packet_loss": -2,
            "probes_sent": 0,
            "results": [],
            "rtt_avg": 0.0,
            "rtt_max": 0.0,
            "rtt_min": 0.0,
            "rtt_stddev": 0.0
        }
    }

Junos:
"item": "4.4.4.4",
                "results": {
                    "error": "Packet loss 100"
                }

**Comment by dbarrosop** [comment]
@jemcek would you mind opening an issue on `napalm-ios`. Looks like the output returned by IOS is not compliant with the model we specified so we need to fix that. Thanks!

@mirceaulinic do you have any news about the "action" model?

**Comment by jemcek** [comment]
done.

**Comment by mirceaulinic** [comment]
No, I don't have any news yet. I guess is going to take longer than I thought...
